docs/topics/request-response.rst

```diff
@@ -607,6 +607,9 @@ Response objects
         This represents the :class:`Request` that generated this response.
     :type request: :class:`Request` object
 
+    :param address: The IP address of the server from which the Response originated.
+    :type address: |twisted.internet.address.IPv4Address|_ / |twisted.internet.address.IPv6Address|_ object
```

Comment:
I don’t think Scrapy supports IPv6 yet (#1031), so this might be misleading.

We will have to remember to mention `twisted.internet.address.IPv6Address` once IPv6 support is implemented, though.

Reply:
Interesting, I wasn't aware of that. I think I can remove the v6 reference and leave a FIXME comment. The links between the issues should also help not to forget about it.